app/src/main/AndroidManifest.xml

```diff
@@ -9,7 +9,7 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/AppTheme" >
-        <activity android:name=".MainActivity" >
+        <activity android:name=".MainActivity" android:configChanges="orientation|screenSize" >

```

Comment:
This is super dangerous, there is probably another way to handle the data that should not rely on overriding configChanges?

From the [docs](https://developer.android.com/guide/topics/resources/runtime-changes#HandlingTheChange):
> Caution: Handling the configuration change yourself can make it much more difficult to use alternative resources, because the system does not automatically apply them for you. This technique should be considered a last resort when you must avoid restarts due to a configuration change and is not recommended for most applications.

Reply:
Removed here https://github.com/power-meter/android-client/pull/32/commits/67e4c8c935c5f1e1c08d38f71487f2132c61b037